File: Python/examples/example3/main.py
""" Defines the main function. 

@returns null
"""
def main():

    """ Input variables x, y.
    """
    x = float(input("Enter a value for x: "))
    y = float(input("Enter a value for y: "))

    """ Their sum.  
    """
    sum = x + y 

    """ Output. 
    """
    if (sum % 2 == 0):
        print ("The sum is even.")
        print ("The sum equals: ", sum)
    else: 
        print ("The sum is odd.")
        print ("The sum equals: ", sum)


""" Runs the main function. 

@returns null 
"""

main()

File: Java/examples/example3/src/java/main.java
package Java.examples.example3.src.java;

class Main{
    public static void main(String[] args) {
        
        int x = 13; 
        int y = 14; 

        int sum = x + y;
        
        if (sum % 2 == 0){

            System.out.println("Their sum is even.");
            System.out.printf("The sum equals: ");
            System.out.println(sum); 

        }
        else{

            System.out.println("Their sum is odd.");
            System.out.printf("The sum equals: ");
            System.out.println(sum); 

        }
    }

}